homework/Contributor.js

```diff
@@ -5,15 +5,28 @@
 // eslint-disable-next-line no-unused-vars
 class Contributor {
   constructor(contributor) {
-    this.contributor = contributor;
+    this.data = contributor;
```

Comment:
Why do you think `this.data` is an improvement over `this.contributor` as was in the original code?

Reply:
because it allows us to attach data of any type to DOM elements in a way that is safe from circular references and therefore from memory leaks.

Reply:
I'm afraid that the name of this property has no relation to anything you mention here.